src/controllers/package.js

```diff
@@ -8,12 +13,47 @@ export const updatePackagesController = (id, price) => {
     return updatePackagesModel(id, price)
 }
 
+export const insertPackageController = async (id, packageDetails) => {
+    const availiable = await getAvailiableSession()
+    if (availiable > 0) {
+        console.log(packageDetails, availiable)
+        await insertPackageModel(id, packageDetails, availiable)
+        return true;
+    }
+    return false;
+}
+
+export const displayRequestPackageController = async () => {
+    return await getRequestpackage()
+}
+
 export const verifyPackagePaymentController = async (id) => {
     const traineeId = await getTraineeIdOfPackage(id)
```

Comment:
Is getting traineeId useful here? If not, you can remove this function call and check.